linkedin_jobs_scraper/linkedin_scraper.py

```diff
@@ -256,7 +256,7 @@ def on(self, event: Events, cb: Callable, once=False) -> None:
         if not isinstance(event, Events):
             raise ValueError(f'Event must be an instance of enum class Events')
 
-        if not isinstance(cb, FunctionType):
+        if not callable(cb):
```

Comment:
FunctionType is now unused. 
Line 3 (Import of FunctionType) should probably be removed